src/tools/ontoweave.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+
+if __name__ == "__main__":
+    import os
+    import sys
+    import logging
+    import argparse
+    import subprocess
+
+    name = os.path.splitext(os.path.basename(sys.argv[0]))[0]
+
+    do = argparse.ArgumentParser(
+        description = "A command line tool to run OntoWeaver mapping adapters on a set of tabular data, and call the created BioCypher export scripts.",
+        epilog = f"Example usage:\n  {name} table.csv:mapping.yaml\n  {name} --config biocypher_config.yaml --schema schema.yaml table1.csv:mapping1.yaml table2.tsv:mapping1.yaml table3.parquet:mapping2.yaml --import-script-run",
+        formatter_class = argparse.RawTextHelpFormatter)
+
+    # {name} --map data_stuff.csv:stuff_mapping.yaml --map data_stiff.csv:stiff_mapping.yaml --config biocypher_config.yaml --schema schema.yaml --call-import-script
+
+    do.add_argument("mapping", metavar="FILE:MAPPING", nargs="+",
```

Comment:
Should this say `--map` ? Like in the example above.

Reply:
I decided to go with a more standard approach, given that at least one mapping is necessary and cannot have a default, hence the positional argument and not an optional one.